TurkishFilter/sections/specific.txt

```diff
@@ -116,6 +116,8 @@ aykutozdemir.com.tr##.theiaStickySidebar > #text-5
 aykutozdemir.com.tr##.theiaStickySidebar > #text-10
 7deniz.net###lock > div[style] > a[href][target="_blank"] > img
 ||7deniz.net/resimler/7_deniz_*_kapak_tek-banner_*.gif
+||kingads.mobi/js/nemp3-tek.js$domain=cepmuzikindir.pro
+cepmuzikindir.pro##a[href*="kingads.mobi"]
```

Comment:
It's kinda wide and a bit out of our syntax in filters. Don't skip useful parts.
`cepmuzikindir.pro##a[href^="https://kingads.mobi/"]`